lib/util/monitored-trip.js

```diff
@@ -21,3 +21,28 @@ export function arrayToDayFields (arrayOfDayTypes) {
   })
   return result
 }
+
+/**
+ * Returns a FormattedMessage string for the pluralised day of week via the react-intl imperative API
+ * such that i18n IDs are hardcoded and can be kept track of by format.js tools
+ */
+export function getFormattedDayOfWeekPlural (day, intl) {
+  switch (day) {
```

Comment:
Here again I would use PropTypes to validate input and make it safe to use template strings to generate the IDs.